pub/lib/dependabot/pub/helpers.rb

```diff
@@ -11,6 +11,22 @@
 module Dependabot
   module Pub
     module Helpers
+      def self.pub_helpers_path
+        File.join(ENV["DEPENDABOT_NATIVE_HELPERS_PATH"], "pub")
+      end
+
+      def self.run_infer_sdk_versions
+        stdout, _, status = Open3.capture3(
+          {},
+          File.join(pub_helpers_path, "infer_sdk_versions")
+        )
+        # TODO(sigurdm): Any good way to log the error here?
+        # puts("Inferring the right Flutter release failed: #{stderr}") unless status.success?
+        return nil unless status.success?
```

Comment:
Would we want the update to attempt to continue if this failed? If it's OK to keep going you could try `Dependabot.logger.warn("Inferring the right Flutter release failed: #{stderr}")`. It's not used much in the codebase but we are looking at increasing its usage in other areas.

Reply:
I first let it fail, but the error messages from the dependency_services tool are much nicer in case of eg. a non-valid `pubspec.yaml` file, so I think it makes sense to continue even if we fail here.